lib/Config.php

```diff
@@ -62,6 +62,28 @@ public function getAllowedTalkGroupIds(): array {
 		return \is_array($groups) ? $groups : [];
 	}
 
+	/**
+	 * @return string[]
+	 */
+	public function getSIPGroups(): array {
+		$groups = $this->config->getAppValue('spreed', 'sip_bridge_groups', '[]');
+		$groups = json_decode($groups, true);
+		return \is_array($groups) ? $groups : [];
+	}
+
+	public function isSIPConfigured(): bool {
+		return $this->getSIPSharedSecret() !== ''
+			&& $this->getDialInInfo() !== '';
+	}
+
+	public function getDialInInfo(): string {
+		return $this->config->getAppValue('spreed', 'sip_bridge_dial-in_info');
```

Comment:
For consistency: `sip_bridge_dial_in_info` or `sip_bridge_dialin_info` (not mixed `-` and `_`).